docs/references/react/use-auth.mdx

```diff
@@ -1,111 +1,118 @@
 
   - `orgSlug`
   - `string`
 
-  The current user's organization slug.
+  The URL-friendly identifier of the user's active organization.
 
   ---
 
   - `signOut()`
   - `(options?: SignOutOptions) => Promise<void>`
 
-  A function that signs the user out. See the [reference documentation](/docs/references/javascript/clerk/clerk#sign-out) for more information.
-
-  ---
+  A function that signs out the current user. Returns a promise that resolves when complete. See the [reference doc](/docs/references/javascript/clerk/clerk#sign-out).
 
   - `getToken()`
   - `(options?: GetTokenOptions) => Promise<string | null>`
 
-  A function that returns a promise that resolves to the current user's session token. Can also be used to retrieve a custom JWT template. See the [reference documentation](/docs/references/javascript/session#get-token) for more information.
+  A function that retrieves the current user's session token or a custom JWT template. Returns a promise that resolves to the token. See the [reference doc](/docs/references/javascript/session#get-token).
 
   ---
 
   - `has()`
   - `(isAuthorizedParams: CheckAuthorizationParamsWithCustomPermissions) => boolean`
 
-  A function that returns a boolean based on the permission or role provided as parameter. Can be used for authorization. See the [reference documentation](/docs/references/nextjs/auth-object#has){{ target: '_blank' }} for more information.
+  A function that checks if the user has specific permissions or roles. See the [reference doc](/docs/references/nextjs/auth-object#has).
 </Properties>
 
 ## How to use the `useAuth()` hook
 
 > [!NOTE]
-> For Next.js apps, it's recommended to use the [`auth()`](/docs/references/nextjs/auth) helper instead of the `useAuth()` hook. Since `auth()` is used in server components, you'll need to pass the auth data to Client Components as needed. If you prefer to use the `useAuth()` hook, pass the `dynamic` prop to the `<ClerkProvider>` component, but note that this switches the app to dynamic rendering. Learn more [here](/docs/references/nextjs/rendering-modes).
+> For Next.js applications, it's recommended to use the [`auth()`](/docs/references/nextjs/auth) helper instead of `useAuth()`. Since `auth()` must be used in Server Components, you'll need to pass auth data to Client Components as needed. If you prefer `useAuth()`, you must pass the `dynamic` prop to `<ClerkProvider>`, but be aware this switches the app to dynamic rendering. Learn more [here](/docs/references/nextjs/rendering-modes).
 
-The following example demonstrates how to use the `useAuth()` hook to access the current auth state, like whether the user is signed in or not. It also demonstrates a basic example of how you could use the `getToken()` method to retrieve a session token for fetching data from an external resource.
+The following example uses the `useAuth()` hook to fetch data from an external API using a session token. The `getToken()` method is used to retrieve the token.
```

Comment:
I still wouldn't say this is correct, as per my last comment:
You don't use useAuth() to fetch data using a session token. You use useAuth() to access the getToken() method, which gives you your session token. Then you can use your session token to fetch data

Reply:
https://github.com/clerk/clerk-docs/pull/1814/commits/d241fcb7a57c0f63d82c4b846d398fc5f850a094